G/GMT/build_tarballs.jl

```diff
@@ -0,0 +1,92 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "GMT"
+version = v"6.4.0"
+
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/GenericMappingTools/gmt", 
+    "19413d486888ebdc3c01aeed3707bcc88eb727df")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+GSHHG_VERSION="2.3.7"
+DCW_VERSION="2.1.1"
+GSHHG="gshhg-gmt-${GSHHG_VERSION}"
+DCW="dcw-gmt-${DCW_VERSION}"
+EXT="tar.gz"
+
+cd $WORKSPACE/srcdir
+cd gmt
+mkdir build
+cd build/
+
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DHAVE_QSORT_R_GLIBC=False \
+    -DHAVE___BUILTIN_BSWAP16=False \
+    -DHAVE___BUILTIN_BSWAP32=False \
+    -DHAVE___BUILTIN_BSWAP64=False \
+    -DGMT_ENABLE_OPENMP=True \
```

Comment:
If you want openmp, you likely want to do https://github.com/JuliaPackaging/Yggdrasil/blob/2cbe1bf44d3c0c1923657f1c6b2aa911d8c685fa/B/blis/build_tarballs.jl#L135-L138 otherwise by default there's no openmp at all for macos: https://buildkite.com/julialang/yggdrasil/builds/1219#0186abbc-f94c-4018-aa49-ad9c7f93c385/639-1057